session.go

```diff
@@ -3024,17 +3024,20 @@ func (c *Collection) Update(selector interface{}, update interface{}) error {
 	}
 	lerr, err := c.writeOp(&op, true)
 	if err == nil && lerr != nil && !lerr.UpdatedExisting {
-		return ErrNotFound
+		return &ChangeInfo{}, ErrNotFound
```

Comment:
You can return the ```info``` from the named return, and it'll be a nil ```ChangeInfo``` in this case instead of an empty one (which makes more sense in case of error). It's also consistent with what's done for the ```Upsert```.